core/src/main/resources/jenkins/install/pluginSetupWizard_it.properties

```diff
@@ -58,21 +57,21 @@ installWizard_installIncomplete_dependenciesLabel=Dipendenze
 installWizard_installingConsole_dependencyIndicatorNote=** - dipendenza obbligatoria
 installWizard_websiteLinkLabel=Sito Web
 installWizard_pluginInstallFailure_title=Errori di installazione
-installWizard_pluginInstallFailure_message=Alcuni plugin non sono stati installati correttamente, � possibile riprovare ad installarli o continuare mantenendo i plugin la cui installazione non � riuscita
+installWizard_pluginInstallFailure_message=Alcuni plugin non sono stati installati correttamente, \u00E8 possibile riprovare ad installarli o continuare mantenendo i plugin la cui installazione non \u00E8 riuscita
 installWizard_continue=Continua
 installWizard_retry=Riprova
 installWizard_upgradePanel_title=Aggiorna
 installWizard_upgradePanel_banner=Benvenuto in Jenkins {0}!
-installWizard_upgradePanel_message=Jenkins {0} include nuove funzionalit� che pensiamo possano piacere, si installino questi plugin aggiuntivi per sfruttarle!
+installWizard_upgradePanel_message=Jenkins {0} include nuove funzionalit\u00E0 che pensiamo possano piacere, si installino questi plugin aggiuntivi per sfruttarle!
 installWizard_upgradePanel_skipRecommendedPlugins=No, grazie
 installWizard_upgradeComplete_title=Aggiorna
 installWizard_pluginsInstalled_banner=Benvenuto in Jenkins {0}!
-installWizard_upgradeComplete_message=Congratulazioni! Si � effettuato l''aggiornamento a Jenkins {0}.</p><p>Per saperne di pi� sulle sue nuove funzionalit�, si visiti <a target="_blank" href="https://jenkins.io/2.0/">il sito Web di Jenkins</a>!
+installWizard_upgradeComplete_message=Congratulazioni! Si \u00E8 effettuato l'aggiornamento a Jenkins {0}.</p><p>Per saperne di pi\u00F9 sulle sue nuove funzionalit\u00E0, si visiti <a target="_blank" href="https://jenkins.io/2.0/">il sito Web di Jenkins</a>!
```

Comment:
l'aggiornamento --> escape?
or you need to rephrase with "Jenkins e' stato aggiornato."